src/Libraries/Microsoft.AspNetCore.HeaderParsing/README.md

```diff
@@ -1 +1,128 @@
+- [ContentDisposition](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.ContentDisposition)
+- [ContentType](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.ContentType)
+- [Cookie](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.Cookie)
+- [Date](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.Date)
+- [IfMatch](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.IfMatch)
+- [IfModifiedSince](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.IfModifiedSince)
+- [IfNoneMatch](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.IfNoneMatch)
+- [IfRange](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.IfRange)
+- [IfUnmodifiedSince](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.IfUnmodifiedSince)
+- [Range](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.Range)
+- [Referer](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames.Referer)
+- XForwardedFor (`X-Forwarded-For`)
+
+## Options
+
+The `HeaderParsingOptions` class is used to configure common behaviors for this feature.
+
+- In case you configured parsing of a specific header, but the header is not present in the current HTTP request, you can pre-configure default values to be returned in such cases. See the `DefaultValues` option.
+- The number of cached values can be changed per header using the `MaxCachedValuesPerHeader` option. If not configured the `DefaultMaxCachedValuesPerHeader` value will be used.
+
+## Metrics
+
+The package generates the following metrics:
+
+| Metric name | Metric value | Dimension | Dimension value |
+|-|-|-|-|
+| `HeaderParsing.ParsingErrors` | Increased by **1** with every parsing error     | `HeaderName` | Header name                                       |
+|                               |                                                 | `Kind`       | Error message returned by the `TryParse()` method |
+| `HeaderParsing.CacheAccess`   | Increased by **1** with every cache hit or miss | `HeaderName` | Header name                                       |
+|                               |                                                 | `Type`       | Either `Hit` or `Miss` for the internal header cache hits and misses respectively |
+
+## Custom headers parsing
+
+Although this package provides support for parsing many HTTP headers, you sometimes need to parse custom headers.
+You can easily extend the header parsing model to support your custom headers by implementing the abstract class `HeaderParser` and then register an 
+instance of this class in the `IHeaderRegistry` interface.
+
+## Feedback & Contributing
+
+For any feedback or contributions, please visit us in [our GitHub repo](https://github.com/dotnet/extensions).
```

Comment:
I kind of liked the "We welcome.." wording I suggested in Chris's PR, because it actively encourages folks to engage not just telling them where ...

Reply:
```suggestion
We welcome feedback and contributions in [our GitHub repo](https://github.com/dotnet/extensions).
```